src/applications/simple-forms/40-0247/tests/e2e/0247-pmc.cypress.spec.js

```diff
@@ -115,6 +116,18 @@ const testConfig = createTestConfig(
           });
         });
       },
+      [pagePaths.certificatesPage]: ({ afterHook }) => {
+        afterHook(() => {
+          cy.get('@testData').then(data => {
+            const { certificates } = data;
+
+            fillTextWebComponent('certificates', certificates);
```

Comment:
reminder that if you use this new property, this page and field should be detected and tested automatically (runs on any page you haven't manually specified)
`useWebComponentFields: true`

Reply:
Prop added.  Thanks!